README.md

```diff
@@ -31,212 +39,299 @@ SCHEMATIC is an acronym for _Schema Engine for Manifest Ingress and Curation_. T
+### 2. Set up your virtual environment
 
-2. **Setting Up the Virtual Environment**
-
-After switching to the version of Python supported by Schematic, please activate a virtual environment within which you can install the package:
+Once you are working with a python version supported by Schematic, please activate a virtual environment within which you can install the package. Python 3 has built-in support for virtual environments with the `venv` module, so you no longer need to install `virtualenv`:
 ```
 python3 -m venv .venv
 source .venv/bin/activate
 ```
-Note: Python 3 has built-in support for virtual environments with the venv module, so you no longer need to install virtualenv.
 
-3. **Installing Schematic**
+### 3. Install `schematic` dependencies
 
 Install the package using [pip](https://pip.pypa.io/en/stable/quickstart/):
 
 ```
 python3 -m pip install schematicpy
 ```
 
-If you run into error: Failed building wheel for numpy, the error might be able to resolve by upgrading pip. Please try to upgrade pip by:
+If you run into `ERROR: Failed building wheel for numpy`, the error might be able to resolve by upgrading pip. Please try to upgrade pip by:
 
 ```
 pip3 install --upgrade pip
 ```
 
-## Installation guide for developers/contributors 
+### 4. Set up configuration files
+
+The following section will walk through setting up your configuration files with your credentials to allow for communication between `schematic` and the Synapse API.
+
+There are two main configuration files that need to be created + modified:
+- `.synapseConfig`
+- `config.yml`
+
+**Create and modify the `.synapseConfig`**
+
+The `.synapseConfig` file is what enables communication between `schematic` and the Synapse API using your credentials.
+You can automatically generate a `.synapseConfig` file by running the following in your command line and following the prompts.
+
+>[!TIP]
+>You can generate a new authentication token on the Synapse website by going to `Account Settings` > `Personal Access Tokens`.
+
+```
+synapse config
+```
+
+After following the prompts, a new `.synapseConfig` file and `.synapseCache` folder will be created in your home directory. You can view these hidden
+assets in your home directory with the following command:
+
+```
+ls -a ~
+```
+
+The `.synapseConfig` is used to log into Synapse if you are not using an environment variable (i.e. `SYNAPSE_ACCESS_TOKEN`) for authentication, and the `.synapseCache` is where your assets are stored if you are not working with the CLI and/or you have specified `.synapseCache` as the location in which to store your manfiests, in your `config.yml` (more on the `config.yml` below).
```

Comment:
Right now the codebase looks for the environment var `SYNAPSE_ACCESS_TOKEN` https://github.com/Sage-Bionetworks/schematic/blob/49c701c1edef7e67eea1ae21216ab9e168c137e5/schematic/store/synapse.py#L369 

I think changing it to `SYNAPSE_AUTH_TOKEN` would lead to access issues on the users' side